scipy/sparse/sputils.py

```diff
@@ -12,7 +12,7 @@
            'isshape', 'issequence', 'isdense', 'ismatrix', 'get_sum_dtype']
 
 supported_dtypes = ['bool', 'int8', 'uint8', 'short', 'ushort', 'intc',
-                    'uintc', 'longlong', 'ulonglong', 'single', 'double',
+                    'uintc', 'int64', 'uint64', 'longlong', 'ulonglong', 'single', 'double',
```

Comment:
I suspect `"l", "L"` should be added here instead of `"int64", "uint64"`. The point is that these are C type codes.

Reply:
@pv `numpy.dtype("l") == numpy.dtype("int64")` and `numpy.dtype("L") == numpy.dtype("uint64")` so it should not make any change if we write `"int64"` instead of `"l"` and `"uint64"` instead of `"L"` .